islandora_solr_config/islandora_solr_config.module

```diff
@@ -44,7 +44,7 @@ function islandora_solr_config_islandora_solr_secondary_display() {
       'class' => 'IslandoraSolrResultsCSV',
       'function' => 'printCSV',
       'description' => 'Download complete resultset as a CSV Spreadsheet',
-      'logo' => '<img src="' . $path . '/images/csv.png" class="secondary-display-csv">',
+      'logo' => '<img src="' . $path . '/images/csv.png" class="secondary-display-csv" alt="CSV Spreadsheet">',
```

Comment:
The human-readable string "CSV Spreadsheet" should be wrapped in `t()`... (and likely the `description` above, but don't feel obliged).

We _may_ want to look at moving this to a `theme('image', [...])` call, but again, not necessary... (I seem to recall there being something not-quite-functional when trying to pass in `class` attributes with Drupal's theme layer which may be an explanation as to why it was not done previously, but I may be mixing up issues)